### PR TITLE
Set initial code owners and fill out some of README.md

### PR DESCRIPTION
**Pull request by dfr**
I don't fully understand how the github maintainer model works - this probably needs other things to change to match my change to CODEOWNERS.

@samuelkarp PTAL.

### CODE REVIEW

**Comment by amye** [comment]
Well, so one thing: "Unknown owner on line 1: make sure @dch exists and has write access to the repository"

**Comment by dfr** [comment]
> Well, so one thing: "Unknown owner on line 1: make sure @dch exists and has write access to the repository"

For the moment, I have removed @dch from CODEOWNERS and I will follow up with him to get the right github handle.

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @amye @caniszczyk
+* @amye @caniszczyk @dfr @samuelkarp @emaste
```
Comment (samuelkarp) [inline CODEOWNERS:1]: I think you can remove Amye and Chris from here.

File: README.md [hunk 2]
```diff
@@ -34,6 +37,10 @@ Details for issues and pull requests are described in the [contributing guidelin
 
 ## Organizers
 
+- Doug Rabson (@dfr)
+- Samuel Karp (@samuelkarp)
+- Ed Maste (@emaste)
+- Dave Cottlehuber (@dch)
 - _Name_ _(@handle)_
```
Comment (samuelkarp) [inline README.md:44]: Let's remove this line.

**Review by samuelkarp — APPROVED** [review]
A couple small changes but otherwise LGTM.

**Review by samuelkarp — APPROVED** [review]

**Comment by dfr** [comment]
Thanks @samuelkarp. I plan to work on defining the requirements for this WG and I've put some initial ideas on an approach to jails [here](https://github.com/opencontainers/wg-freebsd-runtime/blob/simple-jail/docs/proposals/PROPOSAL_A.md).

**Pull request merged**